embedly/gunicorn.conf

```diff
@@ -1,5 +1,6 @@
 import multiprocessing
 
+access_logfile = "-"
```

Comment:
@oyiptong Yep you are correct that will be handled by #29 where we can disable logging at the container level.  The services within the container will log to stdout and then the docker daemon will redirect that output to /dev/null